internal/impl/devbox.go

```diff
@@ -846,11 +852,13 @@ func (d *Devbox) computeNixEnv(ctx context.Context, usePrintDevEnvCache bool) (m
 
 	d.setCommonHelperEnvVars(env)
```

Comment:
I'm unsure if these should go in pure or not. cc: @gcurtis

Reply:
I think we still want `LD_LIBRARY_PATH` and `LIBRARY_PATH`, but without the current environment's values appended. Since it looks like `env["LIBRARY_PATH"]` and `env["LD_LIBRARY_PATH"]` will always be empty, I think this is okay as-is.